routes/user.go

```diff
@@ -1811,12 +1811,87 @@ type GetNotificationsRequest struct {
 	FilteredOutNotificationCategories map[string]bool
 }
 
+type GetNotificationsCountRequest struct {
```

Comment:
nit: It would be great if we could move the "GetNotificationsRequest" and "GetNotificationsResponse" structs back down so that they immediately precede the GetNotfications() handler.  Most endpoint handler functions in the backend repo follow this pattern and it's super convenient when you are reading the code.

Reply:
Structs have been moved